core/src/save/region.rs

```diff
@@ -175,6 +178,21 @@ impl RegionHandle {
                 Biome::from_protocol_id(id).ok_or_else(|| Error::InvalidBiomeId(id))?;
         }
 
+        for entity in &level.entities {
+            match entity {
+                EntityData::Item(item_data) => {
+                    debug!("Found an item entity: {:?}", item_data);
+                    // TODO: Create item entity from data
+                }
+                EntityData::Unknown => {
+                    trace!(
+                        "Chunk {:?} contains an unknown entity type",
```

Comment:
It's unfortunate that serde's `#[serde(other)]` attribute restricts the Unknown entry to being a unit variant, since we can't fetch the stored type of the unknown entity.

Reply:
This should be fine, I think. End users won't care anyway, and eventually all entity types will be implemented.